core/src/main/java/com/linecorp/armeria/common/stream/StreamMessage.java

```diff
@@ -192,6 +193,24 @@
         return of(path, DEFAULT_FILE_BUFFER_SIZE);
     }
 
+    /**
+     * Creates a new {@link StreamMessage} that streams the specified {@link Path}.
+     * The default buffer size({@value PathStreamMessage#DEFAULT_FILE_BUFFER_SIZE}) is used to
+     * create a buffer used to read data from the {@link Path}.
+     * Therefore, the returned {@link StreamMessage} will emit {@link HttpData}s chunked to
+     * size less than or equal to {@value PathStreamMessage#DEFAULT_FILE_BUFFER_SIZE}.
+     *
+     * @param path the path of the file.
+     * @param start the start position of the file to read from; must be non-negative.
+     *              {@code 0} indicates the start of the file.
+     * @param end the end position of the file to read until.
```

Comment:
What do you think of also adding whether the index is exclusive?

```suggestion
     * @param end the end position of the file to read until, exclusive.
```